### DISCUSSION TITLE
Feature Request: Keyboard shortcut to control playback speed of bag like Youtube videos

### DISCUSSION THREAD

**Discussion by davidzimmerjr** [foxglove/community#987] [General]
### Specifically Requested Shortcuts

| Keyboard shortcut | Function |
|--------|--------|
| > | Speed up the ~~video~~ bag file playback rate. |
| < | Slow down the ~~video~~ bag file playback rate. | 

### Reason
Convenience. Sometimes bag files are long or have significant portions of un-useful information that would be nice to quickly play through without using pointer-selection. Likewise, there are highly informative portions of a bag that would be nice to play back slowly.

### Additional Resources

Link to [Youtube's keyboard shortcuts](https://support.google.com/youtube/answer/7631406?hl=en)

**Comment by jtbandes** [comment]
TIL that youtube has a keyboard shortcut for this 😄
